examples/fibs.asm

```diff
@@ -17,6 +17,7 @@ loop:
         addi    t1, t2, 0       // t1 = t2
         addi    s1, s1, 4       // increment storage pointer
         blt     s1, s2, loop    // loop as long as we did not reach array length
+        ebreak
```

Comment:
what is this for? why is the exit gracefully below not enough? I don't actually know what the ebreak instruction does, am curious

Reply:
This is a breakpoint to activate the debugger, I forgot to remove it from the commit, my bad.